draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -568,7 +568,13 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using a JSON-only
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist that aim
+to have similar scope. For example, W3C Verifiable Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines
```

Comment:
'similar scope' sounds as if those other documents also use JSON-only payloads with SD-JWT. I think they can be compared in that they all define credential formats.

Reply:
I updated the text.